nuget/WindowsMobileCenterCrashes.nuspec

```diff
@@ -19,23 +19,23 @@
 
     <!-- UWP -->
     <!-- targets files add correct references to proper platforms -->
-    <file src="nuget/Microsoft.Azure.Mobile.Crashes.targets" target="build/uap10.0"/>
-    <file src="SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.UWP/bin/Reference/Microsoft.Azure.Mobile.Crashes.dll" target="ref/uap10.0" />
+    <file src="$windows_dir$/Microsoft.Azure.Mobile.Crashes.targets" target="build/uap10.0" />
+    <file src="$windows_dir$/Microsoft.Azure.Mobile.Crashes.dll" target="ref/uap10.0" />
     
     <!--x86 -->
-    <file src="SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.UWP/bin/x86/Release/Microsoft.Azure.Mobile.Crashes.dll" target="runtimes/win10-x86/lib/uap10.0" />
-    <file src="Release/WatsonRegistrationUtility/WatsonRegistrationUtility.dll" target="runtimes/win10-x86/native"/>
-    <file src="Release/WatsonRegistrationUtility/WatsonRegistrationUtility.winmd" target="runtimes/win10-x86/lib/uap10.0" />
+    <file src="$windows_dir$/Microsoft.Azure.Mobile.Crashes.dll" target="runtimes/win10-x86/lib/uap10.0" />
+    <file src="$windows_dir$/WatsonRegistrationUtility.dll" target="runtimes/win10-x86/native"/>
```

Comment:
Since the $windows_dir$ doesn't actually include the "x86" "ARM" or "x64" folder, you still need to include it. (See the combined nuspecs for reference)